qiskit_nature/transformers/particle_hole_transformer.py

```diff
@@ -28,5 +45,2102 @@ def transform(self, q_molecule: QMolecule) -> QMolecule:
         Returns:
             A new `QMolecule` instance.
         """
-        # TODO
-        raise NotImplementedError()
+
+        n_qubits = self.num_orbitals * 2
+
+        self._h1 = q_molecule.one_body_integrals
+        self._h2 = q_molecule.two_body_integrals
```

Comment:
Is this not doable then on the integrals that are stored in qmolecule, ie. those directly from the driver like is being done for the active space.

Reply:
We need to use the full one and two body integrals and not the ones stored in the qmolecule. I made sure that the PH transformer is the last to be applied in the  molecular problem transformation